### PR TITLE
WplayBot

### PR DESCRIPTION
**Pull request by Neelaksh-Singh**
## Issue that this pull request solves
This request solves the idea of having a chatbot integrated in our project

Closes: # 128

## Proposed changes
 This is my first time commit , therefore there are no changes.

Brief description of what is fixed or changed

## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface
image

image

image

## Other information

Any other information that is important to this pull request

1) To start conversation the user must chat on this number - +1 415 523 8886
2) To initiate the engine , this code must be entered -  join rocket-jar

[requirements.txt](https://github.com/rpotter12/whatsapp-play/files/4460341/requirements.txt)
[runtime.txt](https://github.com/rpotter12/whatsapp-play/files/4460342/runtime.txt)

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+    reply=fetch_reply(msg, phone_no)
+    # Create reply
+    resp = MessagingResponse()
+    resp.message(reply)
+
+    return str(resp)
+
+if __name__ == "__main__":
+    app.run(debug=True)
```
Comment (rpotter12) [inline app.py:26]: Codacy Issue found: [A Flask app appears to be run with debug=True, which exposes the Werkzeug debugger and allows the execution of arbitrary code.](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5339131)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- app.py  1
- st.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5339131&bid=17477485)

**Comment by xandao6** [comment]
@rpotter12 it doesn’t look like it’s integrated with wplay.

**Comment by Neelaksh-Singh** [comment]
> Please, integrate with wplay project.

how do I integrate it with the project, since i am new to this.
Can u provide me with suitable resources for the same

**Comment by xandao6** [comment]
@Neelaksh-Singh Like we do with the scripts like message_blast

**Pull request closed without merging**